src/main.rs

```diff
@@ -55,6 +55,8 @@ pub struct Args {
     git: String,
     #[structopt(long = "name")]
     name: Option<String>,
+    #[structopt(long = "force", short = "f")]
+    force: bool,
```

Comment:
I decided to name this flag as ```--exact-project-name``` because I couldn't come up with more suitable name than you suggested, and also added help message.